### ISSUE TITLE
Apache  check dies with "list index out of range"

### ISSUE THREAD

**Issue by gmegidish** [DataDog/datadog-agent#4676]
We recently updated datadog-agent, and now apache checks are not working anymore.

Here's my `datadog-agent version`:
```
Agent 6.16.0 - Commit: 3e13b77 - Serialization version: 4.15.0 - Go version: go1.12.9
```

The exception is thrown at 
datadog_checks/apache/apache.py : 115
```
version = raw_version.split('/')[1]
```

Since we have set these in httpd.conf:
```
ServerTokens Prod
ServerSignature Off
```

Our raw_version is "Apache", and not  "Apache/2.4.4"

Thank you!

**Comment by bryankennedy** [comment]
I think that this might be related to this same report over on the datadog-integrations repo:
https://github.com/DataDog/integrations-core/issues/5266

**Comment by olivielpeau** [comment]
@bryankennedy correct, thanks for having a look, this is indeed a duplicate of https://github.com/DataDog/integrations-core/issues/5266, it will be fixed in the upcoming release of the Agent (7.16.1 / 6.16.1), which should be out soon.
@gmegidish thanks for reporting this, closing the issue as a duplicate
